src/pages/Home.jsx

```diff
@@ -0,0 +1,7 @@
+export default function Home() {
+  return (
+    <>
+      <h2 className="home">Wecome to our page!</h2>
+    </>
```

Comment:
- [ ] @otmanTR great work on your component, however, you forgot to add the paragraph section so that your layout matches the wireframe given, also check the nav items separator it would be nice if it matched the wireframe, below is a comparison,


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/221887619-e5c7e11e-29fc-40d5-8923-405d129021a3.png)  |  ![image](https://user-images.githubusercontent.com/61408860/221887704-b49693df-0fa5-4bb0-8ddf-6d5decc74ec1.png)